app/views/shelters/show.html.erb

```diff
@@ -27,6 +27,9 @@
       Rating: <%= review.rating%> <br/>
       Content: <%= review.content %> <br/>
       <%= image_tag "#{review.image_path}", alt: "#{review.image_path}" %>
+      <%= link_to 'Edit Review', "/reviews/#{review.id}/edit" %> <br/>
+      <%= link_to 'Delete Review', "/reviews/#{review.id}", method: :delete %> <br/>
```

Comment:
For styling consistency, can you put these in a `<navbar>` element, still within the `<section>` element? Then they'll inherit the styling you see on the other index pages. Thanks!